modules/exploits/linux/http/goautodial_3_rce_command_injection.rb

```diff
@@ -42,100 +50,114 @@ def initialize(info={})
 
 
   def check
```

Comment:
This `check` method is less error prone. Previously the check method would return `CheckCode::Vulnerable` for every HTTP server so long as the HTTP response body does not contain the string `1421902800`.